frontend/src/components/supp-allowances/SupportNeedsProgrammingAllowance.vue

```diff
@@ -67,25 +67,33 @@
   </v-row>
 
   <v-divider class="mt-2"></v-divider>
-  <v-row no-gutters class="mr-2 my-2">
-    <v-col cols="12">Please describe how you intend to use this funding:</v-col>
-  </v-row>
-  <v-row v-for="item in CHECKBOX_LABELS" :key="item.value" no-gutters>
-    <v-col cols="11" lg="6">
-      <v-checkbox v-model="model.supportFundingModel" density="compact" class="pl-lg-8 mr-0" prepend-icon :value="item.value">
-        <template v-slot:label>
-          <p>
-            {{ item.label }}
-            <v-tooltip v-if="item.tooltip" content-class="tooltip" :text="item.tooltip">
-              <template #activator="{ props }">
-                <v-icon size="large" v-bind="props">mdi-information-slab-circle-outline</v-icon>
-              </template>
-            </v-tooltip>
-          </p>
-        </template>
-      </v-checkbox>
-    </v-col>
-  </v-row>
+  <div v-if="hasInclusionPolicy">
+    <v-row no-gutters class="mr-2 my-2">
+      <v-col cols="12">Please describe how you intend to use this funding:</v-col>
+    </v-row>
+    <v-row v-for="item in SUPPORT_CHECKBOX_LABELS" :key="item.value" no-gutters>
+      <v-col cols="11" lg="6">
+        <v-checkbox v-model="model.supportFundingModel" density="compact" class="pl-lg-8 mr-0" prepend-icon :value="item.value">
+          <template v-slot:label>
+            <p>
+              {{ item.label }}
+              <v-tooltip v-if="item.tooltip" content-class="tooltip" :text="item.tooltip">
+                <template #activator="{ props }">
+                  <v-icon size="large" v-bind="props">mdi-information-slab-circle-outline</v-icon>
+                </template>
+              </v-tooltip>
+            </p>
+          </template>
+        </v-checkbox>
+      </v-col>
+    </v-row>
+  </div>
+  <div v-else>
+    <v-row class="ml-2 my-5">
+      <v-icon size="x-large" color="warning">mdi mdi-alert</v-icon>
+      <h4 class="ml-3">You must have an inclusion policy to apply for Support Needs Funding. Your organization account manager can update inclusion policy details in Account Management.</h4>
```

Comment:
Should Account Management be a hyperlink to the Organization Management page?

Reply:
Probably? The requirements said to link to "account management" - but maybe they mean Org Management?

Reply:
Account Management is a generic name for both Org/Facility Management and User Managment. I see the hyperlink in wireframe so I guess we should have that? It should go to this page: "http://localhost:8082/account-mgmt/manage-organization"

Reply:
FYI, we have a similar hyperlink in ServiceDeliveryView